bin/yii-db-migration.bat

```diff
@@ -2,10 +2,8 @@
 
 @setlocal
```

Comment:
Let's remove it: https://getcomposer.org/doc/articles/vendor-binaries.md#what-about-windows-and-bat-files-

Reply:
done